tests/admin_docs/tests.py

```diff
@@ -261,6 +261,26 @@ def test_method_excludes(self):
         self.assertNotContains(self.response, "<td>set_status</td>")
         self.assertNotContains(self.response, "<td>save_changes</td>")
 
+    def test_methods_with_arguments(self):
+        """
+        Test that methods that take arguments are also displayed
```

Comment:
You may omit the "Tests that" prefix since all tests test things and just describe the expected behavior.